version/1/4/ko/code_of_conduct.md

```diff
@@ -0,0 +1,63 @@
+# 기여자 행동강령 규약
+
+## 서약
+
+개방적이고 친근한 환경을 만들기 위해, 기여자와 메인테이너는 프로젝트와
+커뮤니티에서 연령, 신체, 장애, 인종, 성 정체성과 표현, 경험, 국적, 외모, 인종,
+종교 또는 성적 정체성과 방향에 관계없이 모두 차별없이 참여할 것을 서약합니다.
+
+## 표준
+
+긍정적인 환경을 조성하기 위한 기여자가 해야 할 행동입니다:
+
+* 친근하고 수용적인 언어를 사용하세요
+* 서로 다른 경험과 관점을 존중하세요
+* 건설적인 비판을 수용하세요
+* 무엇이 커뮤니티에 도움이 되는지 생각하세요
+* 다른 커뮤니티 구성원에 대한 공감을 보여주세요
```

Comment:
> 
> @CoralineAda <https://github.com/CoralineAda> Would you say inclusive in "use welcoming and inclusive language" means a general "non-alienating, non-discriminating" or more specifically "non-sexist"?
> 

Definitely non-alienating. It’s the idea that everyone is welcome regardless of how they identify.